src/cloud-code/httpRequest.js

```diff
@@ -1,25 +1,36 @@
 var request = require("request"),
   Parse = require('parse/node').Parse;
 
-var encodeBody = function(body, headers = {}) {
+var encodeBody = function(options = {}) {
```

Comment:
I need to modify the contents of `options`, so desctructuring is not an option right?